### ISSUE TITLE
[Bug]: Fullscreen don't work with android 13

### ISSUE THREAD

**Issue by vanarok** [termux/termux-app#3741]
### Problem description

Gesture pill does not hide in full screen mode enabled through settings.
![Screenshot_20231217-190939_Termux](https://github.com/termux/termux-app/assets/6105977/9784fd20-29dd-4d99-a983-cbad23c87006)

### Steps to reproduce the behavior.

Use these properties: 
fullscreen=true 
use-fullscreen-workaround=true

### What is the expected behavior?

Either hiding the bar or transparent color.https://developer.android.com/develop/ui/views/layout/edge-to-edge
https://developer.android.com/develop/ui/views/layout/edge-to-edge-manually

![Screenshot_20231217-190124_Gallery](https://github.com/termux/termux-app/assets/6105977/98413274-4422-42ee-aba6-8f81b5ae8ecf)
application in which the gesture pill is transparent and the application occupies all available space at the bottom: https://github.com/IacobIonut01/Gallery

### System information

* Termux application version:0.118.0 f-droid
* Android OS version: 13 (aosp, not miui)
* Device model: mi pad 6 pro

**Comment by 2096779623** [comment]
![Screenshot_20240114-102709_Termux](https://github.com/termux/termux-app/assets/57583560/0bfff46a-0c30-4fe8-a3ed-b152b5256936)
When I set `use-fullscreen-workaround` to `true`, there will be an extra gray area above the keyboard.

**Comment by agnostic-apollo** [comment]
@2096779623 Disable margin adjustment in termux app settings.

**Comment by 2096779623** [comment]
> @2096779623 Disable margin adjustment in termux app settings.

![Screenshot_20240114-111920_Pixel Launcher](https://github.com/termux/termux-app/assets/57583560/c911e05c-4d27-463e-9c27-1c6f8502f2d2)
Still exists.

**Comment by agnostic-apollo** [comment]
Did you restart termux app afterwards?

**Comment by 2096779623** [comment]
> Did you restart termux app afterwards?

Yes,I did.

**Comment by agnostic-apollo** [comment]
What if `use-fullscreen-workaround` is `false` and margin adjustment is enabled?

**Comment by 2096779623** [comment]
> What if `use-fullscreen-workaround` is `false` and margin adjustment is enabled?

![Screenshot_20240114-121417_Termux](https://github.com/termux/termux-app/assets/57583560/0bcbed51-43b8-4487-9036-3ce2bd610de1)
Display normally.

**Comment by agnostic-apollo** [comment]
Those two are different implementations to solve similar issues, should only enable one. You seem to have some background at top and are using a custom build, is `fullscreen=true` working as expected?

**Comment by 2096779623** [comment]
> Those two are different implementations to solve similar issues, should only enable one. You seem to have some background at top and are using a custom build, is `fullscreen=true` working as expected?

Yes, I'm using evox and have a "clear" theme set to dark background.
If I set it to default, full screen mode works fine.

![Screenshot_20240114-123236_Termux](https://github.com/termux/termux-app/assets/57583560/2d13d323-a41b-4a1b-84ae-ff28240d663e)
If I just enable `use-fullscreen-workaround` it doesn't change anything.

![Screenshot_20240114-123130_Termux](https://github.com/termux/termux-app/assets/57583560/534c576e-f824-4d10-a4b0-ff442cb33f2e)

**Comment by agnostic-apollo** [comment]
I guess no issue then for your case.
